### PR TITLE
Update igblast to 1.21.0

### PR DESCRIPTION
**Pull request by jcourteau**
Update igblast to 1.21.0

The 1.21.0 version fixes a bug in the osx binary. The bioconda osx package of igblast just wraps the upstream binary, and that uses liblzo2. This was also not relinked correctly in the 1.20 package, resulting in `Library not loaded: /usr/local/opt/lzo/lib/liblzo2.2.dylib` errors.

This is fixed in 1.21; and I've added an explicit dependency against `lzo` for osx. It's not needed on linux, as the package is built from scratch with the lzo functinoality disabled. 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/igblast/meta.yaml [hunk 1]
```diff
@@ -40,10 +40,12 @@ requirements:
     - bzip2
     - libidn11
     - ncbi-vdb >=2.9.6
+    - lzo # [osx]
   run:
     - ncbi-vdb >=2.9.6
     - perl
     - gnutls  # [osx]
+    - lzo
```
Comment (bmcconeghy) [inline recipes/igblast/meta.yaml:48]: @jcourteau do we need to add ` # [osx]` to the run sections as well?

**Comment by jcourteau** [comment]
@BiocondaBot please fetch artifacts

**Comment by jcourteau** [comment]
@BiocondaBot please fetch artifacts

**Comment by jcourteau** [comment]
@BiocondaBot please add label

**Comment by jcourteau** [comment]
We've tested the packages, both linux and osx, and they work as expected. Ready to merge!

**Review by dpryan79 — APPROVED** [review]

**Pull request merged**